docs/guides-using-fake-timers.mdx

```diff
@@ -51,3 +51,22 @@ afterEach(() => {
   jest.useRealTimers()
 })
 ```
+
+## Using fake timers together with `user-event`
```

Comment:
Thanks for the PR @domnantas !
I do think that we shouldn't mention anything about user-event on this page though.
The reason for this is that this a "core" page.

For this PR the changes on this page can be reverted, and

- "This has a side effect - when using fake timers in
your tests, _all_ of the code inside your test uses fake timers." can be a note on this page
- the caution can be moved to https://testing-library.com/docs/user-event/options#advancetimers

Or do you see this differently @MatanBobi ?

Reply:
I 100% agree with you @timdeschryver :)

Reply:
Should I remove
````
`user-event` internally uses `setTimeout` to delay subsequent actions. Using
fake timers in such tests will cause timeouts, since the `setTimeout` callbacks
do not get executed. In order to allow `user-event` to advance the fake timers,
it is necessary to set [`advanceTimers`](user-event/options.mdx#advancetimers)
option in `userEvent.setup()`:

```js
const user = userEvent.setup({advanceTimers: jest.advanceTimersByTime})
```
````
section entirely, or move it somewhere else?

I understand "user-event" and "core" are separate scopes, but when tests break due to fake timers, the `Using fake timers` page in docs is the first thing I checked. Maybe there should at least a note in this page linking to the `advanceTimers` option?
```
:::note

Using fake timers together with `user-event` may cause test timeouts. See [`advanceTimers`](user-event/options.mdx#advancetimers) option for clarification.

:::
```

Reply:
I think it can be removed, or moved to the user-event docs at the advance timers option (but I think it already mentions it).

I like the note 👍.

Reply:
Simplified and moved the explanation to `user-event/options.mdx`. Left a short note on `guides-using-fake-timers`